src/gui/Window.qml

```diff
@@ -100,4 +113,51 @@ ApplicationWindow {
 
         Behavior on scale { HNumberAnimation { overshoot: 3; factor: 1.2 } }
     }
+
+    SystemTrayIcon {
+        property string iconPack: theme ? theme.icons.preferredPack : "thin"
+
+        visible: true
+        tooltip: qsTr("Mirage")
+        icon.source: `../icons/${iconPack}/tray-icon.png`
+
+        onActivated:
+            if (reason !== SystemTrayIcon.Context)
+                window.drawAttention()
```

Comment:
It would be useful to hide/show on left click (instead of only showing) and close on middle click, that way all the menu operations are doable in one click